### PR TITLE
Update troubleshooting.rst

### PR DESCRIPTION
**Pull request by klementuel**
Add issue for Built-in CODE server

### CODE REVIEW

**Comment by tflidd** [comment]
Thanks for adding this. It helps to keep a link to reference the issue. I found this is the official collabora debug page:
https://github.com/nextcloud/all-in-one/discussions/1358

**Review by tflidd — APPROVED** [review]

**Comment by kesselb** [comment]
I find this comment rather confusing then helpful. 

Is it about adding the domain name for the office instance to /etc/hosts?

File: admin_manual/office/troubleshooting.rst [hunk 1]
```diff
@@ -38,4 +38,12 @@ Issue: Collabora Online doesn't handle my 100 users.
...
\ No newline at end of file
+    Yes, this is currently unsupported.
+    
+Issue: Nextcloud Office could not connect to the built-in Collabora Online - Built-in CODE server. 
+    Please check if in /etc/hosts the entry
+    ```
+    127.0.0.1 www.yourdomain.com 
+    ```
+    is entered.
+    if the WOPI warning still appears under Settings/Administration/Office, enter the IP 127.0.0.1 under "Allow list for WOPI requests".
```
Comment (juliushaertl) [inline admin_manual/office/troubleshooting.rst:49]:
```suggestion
Issue: Nextcloud Office could not connect to the built-in Collabora Online - Built-in CODE server. 
    Make sure that the Nextcloud instance is able to reach itself using the same hostname that is used to access through the browser.

    You might want to add your Nextcloud domain to /etc/hosts to ensure the connectivity if DNS resolution doesn't work for this:
    ```
    127.0.0.1 cloud.yourdomain.com 
    ```

    If you continue to see a warning about the WOPI allow list, make sure to also add the IP to the allow list under Settings/Administration/Office.
```

**Comment by juliushaertl** [comment]
Made a suggestion to describe it a bit clearer. @kesselb WDYT?

**Comment by kesselb** [comment]
Awesome @juliushaertl :rocket:

**Review by kesselb — APPROVED** [review]

**Pull request merged**